lib/matplotlib/tests/test_axes.py

```diff
@@ -1150,6 +1150,16 @@ def test_hist_steplog():
     ax = plt.subplot(4, 1, 4)
     plt.hist(data_big, 100, histtype='stepfilled', log=True, orientation='horizontal')
 
+@image_comparison(baseline_images=['hist_step_filled'], extensions=['png'])
+def test_hist_step_empty():
+    import numpy as np
+    import matplotlib.pyplot as plt
+    x = np.random.randn(1000, 3)
```

Comment:
You probably need to set the `random` seed like the test below does for consistency.